servers/rendering/renderer_rd/forward_mobile/scene_shader_forward_mobile.cpp

```diff
@@ -39,6 +39,10 @@ using namespace RendererSceneRenderImplementation;
 
 /* ShaderData */
 
+void SceneShaderForwardMobile::ShaderData::set_path_hint(const String &p_path) {
+	path = p_path;
+}
```

Comment:
Shouldn't this be the default implementation in `ShaderData` to avoid reimplementing it the same way everywhere?